src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Figure implements Drawable {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: " + getClass().getSimpleName()
+                + ", color: " + color.toLowerCase()
+                + ", ";
+    }
+
+    @Override
+    public void draw(Figure figure) {
+        System.out.println(figure);
+    }
+
+    public double getArea() {
+        return 0;
+    }
```

Comment:
```suggestion
```
these methods should be abstract, we let subclasses implement them - we can't know all their details, radius of circle for example, in this abstract superclass